android/src/main/java/com/swmansion/reanimated/nodes/PropsNode.java

```diff
@@ -25,6 +25,28 @@
   private final JavaOnlyMap mPropMap;
   private final ReactStylesDiffMap mDiffMap;
 
+  private void addProp(WritableMap propMap, String key, Object value) {
+    if (value == null) {
+      propMap.putNull(key);
+    } else if (value instanceof Double) {
+      propMap.putDouble(key, (Double) value);
+    } else if (value instanceof Integer) {
```

Comment:
I think that there's no case when it's `Integer` coz JS's number is serialized to `Double`, no?

Reply:
Colors can actually be represented as Integer 
https://github.com/facebook/react-native/blob/0ee5f68929610106ee6864baa04ea90be0fc5160/Libraries/StyleSheet/__tests__/processColor-test.js

https://github.com/facebook/react-native/blob/0ee5f68929610106ee6864baa04ea90be0fc5160/Libraries/StyleSheet/__tests__/normalizeColor-test.js

https://github.com/facebook/react-native/blob/a4cc519a52882a6b08825ecdb15e1c9411c0197d/ReactCommon/fabric/graphics/platform/cxx/Color.h

Also, in react-native-svg:
https://github.com/react-native-community/react-native-svg/blob/master/lib/extract/extractColor.js#L406-L417

Reply:
@msand can you check if the latest version of the patch is in order? We now look at the value type instead of its Java instance.